package.json

```diff
@@ -64,9 +64,11 @@
     "moment": "2.17.1",
     "promise": "7.1.1",
     "react": "15.4.1",
+    "react-addons-shallow-compare": "15.4.1",
```

Comment:
when I installed `react-virtualized` it informed me that this was a required peer dependency but would not be installed automatically. we can try again without it, but I believe that the build will fail.